modules/Planner/units_addProcess.php

```diff
@@ -125,7 +123,7 @@
                     }
 
                     //ADD BLOCKS
-                    $blockCount = ($_POST['blockCount'] - 1);
+                    $blockCount = ($_POST['blockCount'] - 1 ?? 0);
```

Comment:
Not quite there yet. The null coalesce always needs applied directly to the $_POST variable.
```suggestion
                    $blockCount = ($_POST['blockCount'] ?? 0) - 1;
```